### PR TITLE
Hbase

### PR DESCRIPTION
**Pull request by hthayko**
tests fail because of absence of env variables

### CODE REVIEW

**Comment by wilsonianb** [comment]
We should either:
- Modify the tests to not require the environment variable
- Configure circleci to setup an hbase instance (https://github.com/dajobe/hbase-docker)
- Plug the production hbase environment variables in to the circleci project settings

File: package.json [hunk 1]
```diff
@@ -14,6 +14,8 @@
     "chai": "^3.0.0",
     "check-types": "^3.3.0",
     "commander": "^2.8.1",
+    "geoip-lite": "^1.1.6",
```
Comment (wilsonianb) [inline package.json]: This isn't used

File: src/lib/database.js [hunk 2]
```diff
@@ -1,25 +1,27 @@
 'use strict';
-var Sequelize = require('sequelize');
-var sql;
-var sqlInited;
-
+var HBase = require('./hbase-thrift');
+var Promise = require('bluebird');
+var Hbase = Promise.promisifyAll(require('hbase'));
```
Comment (wilsonianb) [inline src/lib/database.js]: `Hbase` doesn't appear to be used

Comment (hthayko) [inline src/lib/database.js]: thanks

File: package.json (hunk 1 above)
Comment (hthayko) [inline package.json]: thanks

File: src/lib/database.js [hunk 3]
```diff
@@ -1,25 +1,26 @@
 'use strict';
-var Sequelize = require('sequelize');
-var sql;
-var sqlInited;
-
+var HBase = require('./hbase-thrift');
+var Promise = require('bluebird');
```
Comment (wilsonianb) [inline src/lib/database.js]: This isn't used :)

Comment (hthayko) [inline src/lib/database.js]: thanks

File: test/utility-test.js [hunk 4]
```diff
@@ -6,7 +6,7 @@ var rc_util = require('../src/lib/utility.js');
 var invalid_crawl = require('./data/invalid_crawl.json');
 var valid_crawl = require('./data/valid_crawl.json');
 
-var db_url = 'postgres://postgres:postgres@127.0.0.1:5432/circle_test';
+var db_url = '52.88.154.28:9090'; // testing engine might not have access to this url.
```
Comment (wilsonianb) [inline test/utility-test.js]: Let's not commit the db_url, especially since we're not using it

Comment (hthayko) [inline test/utility-test.js]: but we are using it in .skipped functions. should I remove skipped functions as well ?

Comment (wilsonianb) [inline test/utility-test.js]: running the tests with `DATABASE_URL` should be fine. We mainly don't need to make that address public

Comment (hthayko) [inline test/utility-test.js]: what do u mean by `DATABASE_URL` ?

Comment (wilsonianb) [inline test/utility-test.js]: sorry, `HBASE_URL`

Comment (hthayko) [inline test/utility-test.js]: how do we provide env vars to circle ci ?

File: test/utility-test.js [hunk 5]
```diff
@@ -107,108 +107,6 @@ describe('Database Util', function() {
       });
     });
   });
-  describe('#getRowById()', function() {
```
Comment (wilsonianb) [inline test/utility-test.js:110]: Should we keep these tests? Is the worker going to be using at least `getRowById`?

Comment (hthayko) [inline test/utility-test.js:110]: getRowById, getRowsByIds and getLatestRow use the same function call under the hood and we have getLatestRow 'tested' already :)

**Comment by wilsonianb** [comment]
LGTM :+1:

**Pull request merged**